src/App.tsx

```diff
@@ -15,36 +17,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPH: number = 1;
+const SORT_BY_LENGTH: number = 2;
+
+const sortGoods = (goods: string[], key: number): string[] => {
+  switch (key) {
+    case SORT_BY_ALPH:
+      return goods.sort((a, b) => a.localeCompare(b));
+    case SORT_BY_LENGTH:
+      return goods.sort((a, b) => a.length - b.length);
+    default:
+      return goods;
+  }
+};
```

Comment:
[CODE STYLE] The function `sortGoods` is not named according to the guidelines. A better name would be `sortGoodsByKey` which clearly indicates what the function does.